### PR TITLE
Disable debugger module selector when no modules available

### PR DESCRIPTION
**Pull request by loosebazooka**

### CODE REVIEW

**Comment by loosebazooka** [comment]
image

@elharo @patflynn this is what this patch makes it look like.

**Comment by loosebazooka** [comment]
closes #228

**Comment by patflynn** [comment]
Cool. Can we hide the ' please select a module' when there are none to select?

**Comment by loosebazooka** [comment]
The validation system kind of drops that in (and it toggles the Ok/Attach button enable/disable). Is there a message you would rather have? It'll start to get real hacky if we take it out.

**Comment by elharo** [comment]
Do we need the PLease select a mopdule label at all? Or maybe it shoudln't be where it is. Instead it should be the default unselectable item in the popup, and is shown only when we're not showing "No debuggable modules found" If you want to do that in a later PR, I'm OK with this PR; but if not I agree withy Patrick that we should hide the label in the case where no module can be selected.

File: core-plugin/resources/messages/CloudToolsBundle.properties [hunk 1]
```diff
@@ -77,7 +77,8 @@ clouddebug.stashbuttontext=Stash local changes
 clouddebug.attach=Attach
 clouddebug.nologin=You must be logged in to perform this action.
 clouddebug.noprojectid=Please enter a Project ID.
-clouddebug.nomodule=Please select a Module.
+clouddebug.nomoduleselected=Please select a Module.
```
Comment (elharo) [inline core-plugin/resources/messages/CloudToolsBundle.properties]: Anyone know the UI standards for capitalization? seems weird to have "Module" here and "module" in the very next line.

Comment (patflynn) [inline core-plugin/resources/messages/CloudToolsBundle.properties]: +katie.

**Comment by patflynn** [comment]
I guess if it's a validation error it should be disabled when the field is
disabled. If it's a label I agree with elliotte that it's redundant and can
be removed.

**Comment by loosebazooka** [comment]
Yeah, it's a strange message, but it's part of the Validation system built into Dialogs into intellij.  You can either 1. have a message and disable the ok button or 2. return null and enable the ok button.

We don't have direct control over it.  I mean we could write our own validation system, but right now we use it control the attach button's enabled-state.

**Comment by patflynn** [comment]
In that case for this particular case maybe it's better to have reinforce
the message that instructs the user what to do in the case of no debuggable
modules.

something like "Please select a project with debuggable modules"

**Comment by patflynn** [comment]
Might be a good place to link to Cloud Debugger docs on deploying
debuggable modules..

**Comment by loosebazooka** [comment]
image

here's what it looks like now

**Comment by patflynn** [comment]
ship it!

**Comment by loosebazooka** [comment]
is that an lgtm?

**Comment by patflynn** [comment]
lgtm.

**Pull request merged**